test/e2e/apps/replica_set.go

```diff
@@ -340,3 +348,63 @@ func testRSAdoptMatchingAndReleaseNotMatching(f *framework.Framework) {
 	})
 	framework.ExpectNoError(err)
 }
+
+func testRSScaleSubresources(f *framework.Framework) {
+	ns := f.Namespace.Name
+	c := f.ClientSet
+
+	// Create webserver pods.
+	rsPodLabels := map[string]string{
+		"name": "sample-pod",
+		"pod":  WebserverImageName,
+	}
+
+	rsName := "test-rs"
+	replicas := int32(1)
+	ginkgo.By(fmt.Sprintf("Creating replica set %q that asks for more than the allowed pod quota", rsName))
+	rs := newRS(rsName, 1, rsPodLabels, WebserverImageName, WebserverImage, nil)
```

Comment:
2nd parameter is `replicas` right?

Reply:
let's use the variable instead of the hard coded `1`

Reply:
Yes it is.

Reply:
Yes, I should have used the var. Updated 
Thank you.